File: xmlws-front/src/app/dto/AccommodationUnitSearchDTO.ts
export class AccommodationUnitSearchDTO{
    _startDate: Date;
    _endDate: Date;

    get startDate() : Date {
        return this._startDate;
    } 

    set startDate(theStartDate: Date) {
        this._startDate = theStartDate;
    }

    get endDate() : Date {
        return this._endDate;
    } 

    set endDate(theEndDate: Date) {
        this._endDate = theEndDate;
    }
}

File: TicketSales/src/main/java/ftn/xmlws/dto/AccommodationUnitSearchDTO.java
package ftn.xmlws.dto;

import java.time.LocalDate;

public class AccommodationUnitSearchDTO {

	private LocalDate startDate;
	private LocalDate endDate;
	
	public AccommodationUnitSearchDTO() {
		
	}

	public LocalDate getStartDate() {
		return startDate;
	}

	public void setStartDate(LocalDate startDate) {
		this.startDate = startDate;
	}

	public LocalDate getEndDate() {
		return endDate;
	}

	public void setEndDate(LocalDate endDate) {
		this.endDate = endDate;
	}
}
